### PR TITLE
Add Paypal donation button

### PR DESCRIPTION
**Pull request by viroulep**
When discussing on Slack @EdHollingdale told me we did have a paypal donate page, but that it needed to be added to the website.

### CODE REVIEW

**Comment by viroulep** [comment]
Forgot the screenshot:
paypal

**Comment by Mollerz** [comment]
Looks good to me! Thanks for setting this up 😁

File: WcaOnRails/app/views/static_pages/about.html.erb [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <hr class="my-4">
 
     <a class="btn btn-primary btn-lg" href="https://www.nfggive.com/guidestar/82-3825954" role="button"><%= t('about.donation_banner.donate_credit') %></a>
```
Comment (jonatanklosko) [inline WcaOnRails/app/views/static_pages/about.html.erb:9]: Here and in `en.yml` we have this analogous URL to Guidestar, could you unify that to match what you did for Paypal?

**Review by jonatanklosko — APPROVED** [review]
Perfect, one suggestion!

File: WcaOnRails/app/views/static_pages/about.html.erb (hunk 1 above)
Comment (viroulep) [reply WcaOnRails/app/views/static_pages/about.html.erb]: Sure thing, I've made the change and rebased the PR!

Comment (jonatanklosko) [reply WcaOnRails/app/views/static_pages/about.html.erb]: Perfect, thank you!

**Pull request merged**